<?xml version="1.0"?>
<launch>

  <node name="traj_tracking" pkg="bring_up" type="traj_tracking.py" output="screen">


    <!-- Map for random 32-by-32 -->
    <param name="scale" type="double" value="18.6" />
    <param name="traj_load_path" value="$(find bring_up)/trajectory/traj_data_random32/solTraj_data_13.txt" />
    <param name="traj_save_path" value="$(find bring_up)/trajectory/traj_data_random32/recTraj_terrain_data_13_open.txt" />


    <param name="time_step" type="double" value="0.1" />
  </node> 



</launch>
